lib/components/src/controls/Number.stories.tsx

```diff
@@ -6,22 +6,22 @@ export default {
   component: NumberControl,
 };
 
-export const Basic = () => {
-  const [value, setValue] = useState(10);
+const Template = (initialValue) => {
+  const [value, setValue] = useState(initialValue);
   return (
     <>
       <NumberControl name="number" value={value} onChange={(newVal) => setValue(newVal)} />
-      <p>{value}</p>
+      <pre>{JSON.stringify(value) || 'undefined'}</pre>
     </>
   );
 };
 
-export const Undefined = () => {
-  const [value, setValue] = useState(undefined);
-  return (
-    <>
-      <NumberControl name="number" value={value} onChange={(newVal) => setValue(newVal)} />
-      <p>{value}</p>
-    </>
-  );
-};
+export const Basic = () => Template(10);
+
+export const Zero = () => Template(0);
```

Comment:
What about decimals?

Reply:
They work, but sure, we should add a story.